client-java/src/rpc/RequestBuilder.java

```diff
@@ -46,7 +45,7 @@
      */
     public static class Transaction {
 
-        public static SessionProto.Transaction.Req open(grakn.core.Keyspace keyspace, GraknTxType txType) {
+        public static SessionProto.Transaction.Req open(grakn.core.Keyspace keyspace, grakn.core.Transaction.Type txType) {
```

Comment:
maybe we can change these to be imports in next PR?

Reply:
👍